tools/depends/native/TexturePacker/Makefile

```diff
@@ -10,7 +10,6 @@ endif
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
```

Comment:
I'd like to keep this as is

Reply:
> I'd like to keep this as is

```
g++ -DTARGET_POSIX  -std=c++0x -g -O2 -llzo2 -ljpeg -lgif  -lpng16 -lm -lm -lz  -static  -o TexturePacker md5.o XBTFWriter.o TexturePacker.o DecoderManager.o decoder/PNGDecoder.o decoder/JPGDecoder.o decoder/GifHelper.o decoder/GIFDecoder.o XBTF.o  -llzo2 -ljpeg -lgif  -lpng16 -lm -lm -lz 
/usr/bin/ld: cannot find -llzo2: No such file or directory
/usr/bin/ld: cannot find -ljpeg: No such file or directory
/usr/bin/ld: cannot find -lgif: No such file or directory
/usr/bin/ld: cannot find -lpng16: No such file or directory
collect2: error: ld returned 1 exit status
```

So either we expect distros to have the static libs or the linking is broken.

Reply:
this makefile is for depends builds, not for distros, static libs are provided:
https://github.com/xbmc/xbmc/blob/master/tools/depends/native/Makefile#L76

Reply:
So we should just remove the entire block for 

```
ifeq ($(NATIVEPLATFORM),)
```